### PR TITLE
Fix error when running `wazuh-install.sh` after canceling the execution

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|#2024|

## Description

This PR fixes an error in the wazuh-install.sh script that caused it to fail if we had previously canceled the execution of the script.

The error could appear because of two reasons:
1. Some files from the wazuh-indexer were not deleted in the rollback function.
2. After canceling the execution some packages were installed but the script did not know it.

### CODE REVIEW

File: unattended_installer/install_functions/installCommon.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # Foundation.
 
 function installCommon_cleanExit() {
-
+    set -x
```
Comment (verdx) [inline unattended_installer/install_functions/installCommon.sh:10]: Is this debug or necessary?

**Review by verdx — APPROVED** [review]
lgtm

File: unattended_installer/install_functions/installCommon.sh (hunk 1 above)
Comment (DFolchA) [reply unattended_installer/install_functions/installCommon.sh:10]: Removed

**Review by davidcr01 — APPROVED** [review]
GJ. Please, assign yourself to the PR.

**Pull request merged**